osu.Game.Rulesets.Osu/Edit/DrawableOsuEditRuleset.cs

```diff
@@ -58,8 +69,17 @@ private void updateState(DrawableHitObject hitObject, ArmedState state)
 
                     case DrawableHitCircle circle: // also handles slider heads
                         circle.ApproachCircle
-                              .FadeOutFromOne(editor_hit_object_fade_out_extension)
+                              .FadeOutFromOne(editor_hit_object_fade_out_extension * 4)
                               .Expire();
+
+                        circle.ApproachCircle.ScaleTo(1.1f, 300, Easing.OutQuint);
+
+                        var circlePieceDrawable = circle.CirclePiece.Drawable;
+
+                        // clear any explode animation logic.
+                        circlePieceDrawable.ApplyTransformsAt(circle.HitStateUpdateTime, true);
+                        circlePieceDrawable.ClearTransformsAfter(circle.HitStateUpdateTime, true);
```

Comment:
This logic can alternatively be moved inside the `Animate` methods of both `CirclePiece`s and we can call

```csharp
(circlePieceDrawable as IMainCirclePiece)?.Animate(ArmedState.Miss);
```

I had this previously and not sure which I prefer.

Reply:
I prefer it this way, just to keep these overrides all in one place.